collections/Lat_th/MS_Lat_th_e_22.xml

```diff
@@ -51,132 +46,176 @@
                      <binding contemporary="true">
-                        <p>Contemporary red sheepskin binding over boards.</p>
+                        <p>Contemporary whittawed leather binding, dyed red, over wooden boards. Metal clasp.</p>
                      </binding>
                   </bindingDesc>
-                  <accMat>The paste down on the inside front cover and folios 133 and 134 consist of leaves of a 13th cent. MS. containing (fol.133, 133v) part of the ‘Exhortatio ad presbyteros’ ascribed to pope <persName role="aut" key="person_58608059">
-                        Eutychianus
-                     </persName> (P.L. 5, col.163–167B, ‘propter quosdam infideles’), part of the
-                     
-                     Corrector of <persName role="aut" key="person_111045814">
-                        Burchard of Worms
-                     </persName> (fol.133v) (P.L. 140, col.950C ‘sacerdos irroget penitentem’- col.951C ‘si fecisti’), and (fol.134) (P.L.col.981A ‘leguminibus siccis’ - 983D ‘pater noster decantet’) and (pastedown and fol.134v) unidentified penitential pieces. The section on fol.134v 
-                     beg: ‘Is qui penitenciam’ is causa xxvi Quaestio VI, ch. 8 of <persName role="aut" key="person_33242385">
-                        Gratian
-                     </persName>'s Decretum.</accMat>
                </physDesc>
-               
                <history>
                   <origin>
                      <origDate calendar="Gregorian" when="1392">1392</origDate>
-                     <origPlace cert="low">
-                        <country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement> (?)<!--ORIGINAL: German, Erfurt (?)--></origPlace>
-                  </origin>
-                  <provenance>'Iste liber est <orgName key="org_415145858126223022295"><!-- check? -->
-                        cartusien. prope Erfford
-                     </orgName>', 15th cent. (fol. 1)</provenance>
-                  <provenance>Perhaps
-                     in the <orgName key="org_415145858126223022295" cert="low">
-                        Salvatorberg charterhouse
-                     </orgName> catalogue, 15th century (as 0 38 or 39) (P. Lehmann, <title>Mittelalt. Bibliothekakataloge Deutschlands</title> II, 496)</provenance>
-                  <provenance>Bought 1935.</provenance>
+                     <origPlace><country key="place_7000084">German</country>, <settlement key="place_7024031">Erfurt</settlement>, <orgName key="org_415145858126223022295">Charterhouse of Salvatorberg</orgName></origPlace>: <quote>Iste liber est cartusien’ prope Erfford’</quote> (fol. 1r, 15th century); <quote>est Carthus’ prope Erffordiam</quote> (fol. 1r, 15th century); <quote>Est Carthusien’ prope Erffordiam</quote> (text hand, fol. 132v). Probably 038 or 39 in the catalogue by P. Lehmann, <title>Mittelalt. Bibliothekakataloge Deutschlands</title> II, 496.</origin>
```

Comment:
Is there evidence for it being written at the Charterhouse though? Otherwise I think the formula in the existing description is preferable.

Reply:
Watson was uncertain, but he didn't notice the note on fol. 132v, which looks to be in the text hand.